docs/modules/servers/pages/postgresql/run/run-docker.adoc

```diff
@@ -0,0 +1,145 @@
+= Postgresql James Server &mdash; Run with docker
+:navtitle: Run with docker
+
+== Running via docker-compose
+
+Requirements: docker & docker-compose installed.
+
+When you try James this way, you will use the most current state of James.
+
+=== Running with Postgresql only
+
+It will be configured to run with Postgresql.
+All those components will be started with a single command.
+
+You can retrieve the docker-compose file : ( docker-compose file and james image name should be changed)
+
+    $ wget https://raw.githubusercontent.com/apache/james-project/master/server/apps/postgres-app/docker-compose.yml
+
+
+Then, you just have to start the services:
+
+    $ docker-compose up -d
+
+Wait a few seconds in order to have all those services start up. You will see the following log when James is available:
+james           | Started : true
+
+A default domain, james.local, has been created. You can see this by running:
+
+    $ docker exec james james-cli -h 127.0.0.1 -p 9999 listdomains
+
+James will respond to IMAP port 143 and SMTP port 25.
+You have to create users before playing with james. You may also want to create other domains.
+Follow the 'Useful commands' section for more information about James CLI.
```

Comment:
I see no reference to the 'Useful commands'.

the same for distributed docs https://james.staged.apache.org/james-distributed-app/3.8.1/run/run-docker.html